lib/packages.js

```diff
@@ -82,8 +82,12 @@ api.readPackagesSync = rootModule => {
   // sort packages according to dependencies
   _packageCache = _sortPackages(_packageCache);
 
-  Object.keys(_packageCache).forEach(
-    name => _getPackageOverrides(_packageCache[name]));
+  // manifest override must happen here to be applied to compile-less`
```

Comment:
there's a trailing backtick here ... dunno if another is missing or if that one should be removed